.github/workflows/build.yml

```diff
@@ -18,7 +18,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        ruby: [2.6, 2.7, '3.0', 3.1]
+        ruby: [2.7, '3.0', 3.1]
```

Comment:
While we're at it, you want to go ahead and add `3.2` to the list since it's an LTS version now?

Reply:
Will do!